shared/tracker2/remote-proxy.desktop.tsx

```diff
@@ -18,15 +18,18 @@ const RemoteTracker = (props: {trackerUsername: string}) => {
   const details = Constants.getDetails(state, trackerUsername)
   const {infoMap} = state.users
   const {avatarRefreshCounter, following, followers, httpSrvToken, httpSrvAddress, username} = state.config
-  const {assertions, bio, blocked, followersCount, followingCount, fullname, guiID} = details
+  const {assertions, bio, followersCount, followingCount, fullname, guiID} = details
   const {hidFromFollowers, location, reason, teamShowcase} = details
   const {counts, errors} = state.waiting
+  const {blockMap} = state.users
```

Comment:
could combine this with the infoMap declaration, `const {infoMap, blockMap} = state.users`